React/Views/ScrollView/RCTScrollView.m

```diff
@@ -299,6 +299,12 @@ - (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher
     }
 #endif
 
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000 /* __IPHONE_13_0 */
```

Comment:
I think it is, and it seems to be a convention in RN project.

`@available(iOS 13.0, *)` is a runtime check, and `#if ... __IPHONE_OS_VERSION_MAX_ALLOWED` check is for SDK the project is compiled against.

I can re-check but I think if you tried to compile this code in Xcode 10.2 or earlier, it would fail to compile

Reply:
I believe @radex may be referring to RCTAppearance.mm where I used this pattern as Dark Mode is new to iOS 13: https://github.com/facebook/react-native/blob/aff6bad27c6c2232ba8bde17823d0a0db4ac589b/React/CoreModules/RCTAppearance.mm#L28-L29

@sammy-SC do you still want this change to be made?

Reply:
@hramos is there any reason to have conditional compilation there? If not, then let's remove it.